test/templates/custom-script_sample.template

```diff
@@ -1,3 +1,5 @@
 aws s3 cp s3://ec2-instance-qualifier-app/ec2-instance-qualifier-app .
 chmod u+x ec2-instance-qualifier-app
-./ec2-instance-qualifier-app >/dev/null 2>/dev/null &
\ No newline at end of file
+./ec2-instance-qualifier-app >/dev/null 2>/dev/null &
+
+sudo yum -y install jq
```

Comment:
you can drop the sudo if this is in the userdata script since it runs that script as root anyways.